### ISSUE TITLE
F

### ISSUE THREAD

**Issue by rodinux** [YunoHost-Apps/monitorix_ynh#42]

**Comment by rodinux** [comment]
Hello,
I have tried to install monitorix, but something go wrong !
the logs here https://paste.yunohost.org/raw/padirecejo

```
2022-12-16 22:11:23,497: DEBUG - + ynh_die '--message=https://www.monitorix.org/monitorix_3.14.0-izzy1_all.deb:
2022-12-16 22:11:23,497: DEBUG - 2022-12-16 22:11:23 ERROR 404: Not Found.'
2022-12-16 22:11:23,498: DEBUG - 2022-12-16 22:11:23 ERROR 404: Not Found.'
2022-12-16 22:11:23,515: DEBUG - 2022-12-16 22:11:23 ERROR 404: Not Found.'
2022-12-16 22:11:23,516: DEBUG - + ynh_exit_properly
2022-12-16 22:11:23,516: WARNING - https://www.monitorix.org/monitorix_3.14.0-izzy1_all.deb:
2022-12-16 22:11:23,516: WARNING - 2022-12-16 22:11:23 ERROR 404: Not Found.
2022-12-16 22:11:25,019: ERROR - Unable to install monitorix: An error occurred inside the app installation script
```

**Comment by Josue-T** [comment]
Should be fixed by #44
